tests/infra/network.py

```diff
@@ -897,6 +897,22 @@ def wait_for_new_primary(self, old_primary, nodes=None, timeout_multiplier=2):
         logs = []
         while time.time() < end_time:
             try:
+                backup = self.find_any_backup()
+                if backup.get_consensus() == "bft":
+                    try:
+                        with backup.client("user0") as c:
+                            _ = c.post(
+                                "/app/log/private",
+                                {
+                                    "id": -1,
+                                    "msg": "This is submitted to force a view change",
+                                },
+                            )
+                        time.sleep(5)
+                    except CCFConnectionException:
```

Comment:
I would do another backup = self.find_any_backup() here in the hope to get a happier backup, and perhaps shorten the sleep to 1?